paymentsheet/src/test/java/com/stripe/android/utils/WtfComposeTestRule.kt

```diff
@@ -0,0 +1,43 @@
+package com.stripe.android.utils
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.test.SemanticsNodeInteraction
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import org.junit.rules.TestRule
+import org.junit.rules.TestWatcher
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+/**
+ * Using this TestRule to fix tests that we're hanging for inexplicable reasons.
+ */
+class WtfComposeTestRule : TestRule, TestWatcher() {
```

Comment:
```suggestion
class HackyComposeTestRule : TestRule, TestWatcher() {
```

I'm fine with leaving it too.